docs/guidance_for_upgraders.md

```diff
@@ -0,0 +1,69 @@
+# Guidance for Operator Developer
+
+After you change either the operator signature in a BC breaking way or the semantics of the operator, you will need to write an “upgrader” to make the change non-BC breaking. In general, you can know your operator is BC breaking, if it fails `test/forward_backward_compatibility/check_forward_backward_compatibility.py `
```

Comment:
This is not always true. Prototype and beta modules don't have BC-breaking guarantees and upgraders are only required if the operator is actually used by a torchscript or mobile model.

Reply:
Ah, good to know. What is an example of such a module? Is there a canonical source of truth where we can query whether a module is beta and not dispatchable to TorchScript?